showcase/src/app/welcome/components/news-item/_news-item-theme.scss

```diff
@@ -0,0 +1,42 @@
+.section-border {
```

Comment:
Can you move all the properties that are not "themable" to the `xxx-component.scss` file?